packages/fxa-auth-server/lib/senders/emails/templates/subscriptionAccountReminderFirst/index.stories.ts

```diff
@@ -0,0 +1,23 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Meta } from '@storybook/html';
+import { subplatStoryWithProps } from '../../storybook-email';
+
+export default {
+  title: 'SubPlat Emails/Templates/subscriptionAccountReminderFirst',
+} as Meta;
+
+const createStory = subplatStoryWithProps(
+  'subscriptionAccountReminderFirst',
+  'Sent when a user needs to finish setting up their account.',
```

Comment:
I think this could use a little more detail too. I think this is sent whenever a user signs up through the subplat passwordless flow and ignores the other email you have up in a PR right, and then they're sent this reminder to finish setting up the account?